doc/user/content/sql/create-connection.md

```diff
@@ -227,12 +227,18 @@ Field                                   | Value            | Required | Descript
 ----------------------------------------|------------------|:--------:|-------------------------------
 `connection`                            | object name      | ✓        | The name of an [AWS PrivateLink connection](#aws-privatelink) through which network traffic for this broker should be routed.
 `PORT`                                  | `integer`        |          | The port of the AWS PrivateLink service to connect to. Defaults to the broker's port.
+`ssh_connection`                        | object name      | ✓        | The name of an [SSH tunnel connection](#ssh-tunnel) through which network traffic for this broker should be routed.
 
-The `USING PRIVATELINK` clause specifies that Materialize should connect to the
-designated broker in a Kafka cluster via an AWS PrivateLink service.
+The `USING` clause specifies that Materialize should connect to the
+designated broker in a Kafka cluster via an SSH bastion or
+an AWS PrivateLink service. Brokers need not be configured the same way.
 
+{{< warning >}}
 You must attach the clause individually to each broker that you want to connect
-to via AWS PrivateLink.
+to via the tunnel. If there are other broker hosts discovered that are not
+specified, materialize will attempt to connect to them directly.
+{{< /warning >}}
```

Comment:
```suggestion
to via the tunnel.

{{< warning >}}
```suggestion
If your Kafka cluster advertises brokers that are not specified
in the `BROKERS` clause, Materialize will attempt to connect to
those brokers without any tunneling.
```
{{< /warning >}}
```